extensions/DynamicPageList/DPLSetup.php

```diff
@@ -1425,7 +1425,7 @@ private static function isRegexp ($needle) {
         return false;
     }
 
-    public static function dplReplaceParserFunction(&$parser, $text, $pat, $repl='') {
+    public static function dplReplaceParserFunction($parser, $text, $pat, $repl='') {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$parser". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)